probe_scraper/probe_expiry_alert.py

```diff
@@ -5,21 +5,21 @@
 from collections import defaultdict
 import argparse
 import datetime
+import logging
 
 import requests
 
 from probe_scraper import emailer
-from probe_scraper.scrapers import release_calendar
+from probe_scraper.parsers.utils import get_major_version
 
 FROM_EMAIL = "telemetry-alerts@mozilla.com"
 DEFAULT_TO_EMAIL = "dev-telemetry-alerts@mozilla.com"
-EMAIL_DAYS_BEFORE = 14
 PROBE_INFO_BASE_URL = "https://probeinfo.telemetry.mozilla.org/"
 
 EMAIL_BODY_FORMAT_STRING = """
-The following Firefox probes will expire in the next {expire_days} days or have already expired.
+The following Firefox probes have either expired or will expire in the next major Firefox nightly release: version {next_version}.
 
-{per_channel_probes}
+{probes}
 
 What to do about this:
 1. If one, some, or all of the metrics are no longer needed, please remove them from their definitions files (Histograms.json, Scalars.yaml, Events.yaml).
```

Comment:
Huh, missed this last time, but we have a `[1]` footnote for data collection review, but we don't reference it in "What to do about this" number 2.